project-proposals/stage-2/stage-2-questionnaire.md

```diff
@@ -0,0 +1,60 @@
+# Stage 2: Incubation
+
+*As a project intending to move to stage 2 of the OpenRail incubation process, please fill out this questionnaire and send it to the Technical Committee. See the [incubation process](../../incubation-process.md) for details about criteria and requirements for new projects and how it works to get new projects into the association.*
+
+*Copy this template and fill in your answers to the questions in the sections below.*
+
+## Health metrics
+
+*How to assess project health depends on the context of the concrete project. Please answer the questions below and add any additional explanations or data you see as useful to show that the project has a healthy community and development activity.*
+
+### How many committers does the project have?
+
+
+### How many different organizations are contributing to the project?
+
+
+### How much is the project dependent on individual contributors?
+
+*How high is the risk to a project should the most active people leave? This is called the [Contributor Absence Factor](https://chaoss.community/kb/metric-contributor-absence-factor/), sometimes also referred to as bus factor.*
+
+
+### How active is the project?
+
+*Indicate activity by number of commits, number of issues closed, pull request merged, etc.*
+
+
+## Open governance
+
+### What is the release process of the project?
+
+*Indicate documentation of the release process, how changes are communicated, versioning schemes, signing of release artifacts, etc.*
+
+
+### How are decisions taken in the project?
```

Comment:
> [!NOTE]
> Not sure about this one...

```suggestion
### How are decisions made in the project?
```

Reply:
I guess you are right. It's the more common phrase, especially in American English (see also https://english.stackexchange.com/questions/6431/what-is-the-difference-between-make-decision-and-take-decision).